package.json

```diff
@@ -2,59 +2,63 @@
   "name": "@americanexpress/lumberjack",
   "version": "1.1.3",
   "description": "Lumberjack is a minimal, configurable Console with utilities.",
-  "main": "lib/index.js",
-  "scripts": {
-    "test:lint": "eslint --ignore-path .eslintignore --ext .js,.snap .",
-    "test:unit": "jest",
-    "test": "npm run test:unit && npm run test:lint",
-    "build": "babel src -d lib",
-    "prepack": "npm run test && npm run build"
-  },
-  "devDependencies": {
-    "@commitlint/cli": "^8.2.0",
-    "@commitlint/config-conventional": "^8.2.0",
-    "@semantic-release/changelog": "^3.0.5",
-    "@semantic-release/commit-analyzer": "^6.3.2",
-    "@semantic-release/git": "^7.0.18",
-    "@semantic-release/npm": "^5.3.4",
-    "@semantic-release/release-notes-generator": "^7.3.2",
-    "amex-jest-preset": "^5.0.0",
-    "babel-cli": "^6.26.0",
-    "babel-preset-env": "^1.6.1",
-    "eslint": "^4.16.0",
-    "eslint-config-amex": "^7.1.0",
-    "husky": "^3.0.9",
-    "jest": "^23.6.0",
-    "semantic-release": "^15.13.31"
-  },
-  "jest": {
-    "preset": "amex-jest-preset"
-  },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/americanexpress/lumberjack.git"
-  },
   "keywords": [
     "logging",
     "format",
     "logs",
     "log"
   ],
+  "license": "Apache-2.0",
+  "homepage": "https://github.com/americanexpress/lumberjack",
+  "bugs": {
+    "url": "https://github.com/americanexpress/lumberjack/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/americanexpress/lumberjack.git"
+  },
   "contributors": [
     "Andres Escobar <Andres.Escobar@aexp.com> (https://github.com/anescobar1991)",
-    "James Singleton <James.Singleton1@aexp.com> https://github.com/JamesSingleton)",
+    "James Singleton <James.Singleton1@aexp.com> (https://github.com/JamesSingleton)",
     "Jimmy King  <Jimmy.King@aexp.com> (https://github.com/10xLaCroixDrinker)",
     "Jonathan Adshead <Jonathan.Adshead@aexp.com> (https://github.com/JAdshead)",
     "Michael Tobia <Michael.M.Tobia@aexp.com> (https://github.com/Francois-Esquire)",
-    "Michael Tomcal <Michael.A.Tomcal@aexp.com> (https://github.com/mtomcal))",
+    "Michael Tomcal <Michael.A.Tomcal@aexp.com> (https://github.com/mtomcal)",
     "Stephanie Coates  <Stephanie.Coates1@aexp.com> (https://github.com/stephaniecoates)",
     "Nelly Kiboi <Nelly.J.Kiboi@aexp.com> (https://github.com/nellyk)",
     "Nickolas Oliver <nickolas.oliver@aexp.com> (https://github.com/PixnBits)"
   ],
-  "license": "Apache-2.0",
+  "main": "lib/index.js",
+  "module": "src/index.js",
```

Comment:
I see [main](https://docs.npmjs.com/files/package.json.html#main) and [browser](https://docs.npmjs.com/files/package.json.html#browser) but not "module". Node.js uses [type](https://nodejs.org/api/esm.html#esm_package_json_type_field) with [main](https://nodejs.org/api/esm.html#esm_package_json_main). What is "module" used for? 

(If for browsers via bundlers, is this library designed for use in the browser?)

Reply:
Ah, rollup and webpack use that field, Node.JS is the one using something different
https://github.com/rollup/rollup/wiki/pkg.module

Reply:
So `module` is used to resolve ES modules when resolving an npm package (during bundling usually). So yeah, `rollup` and `webpack` use this field on the `package.json` to decide entry for a package. Here is an example using `webpack`:

https://webpack.js.org/configuration/resolve/#resolvemainfields

Reply:
Good point with the browser - currently, the library is node-centric and written like that, we could use the `browser` field in the future for an output that limits usage to the browser environment.